doc/_ext/ceph_commands.py

```diff
@@ -156,31 +154,37 @@ def is_reasonably_simple(self):
 
     def mk_bash_example(self):
         simple = self.is_reasonably_simple()
-        line = ' '.join(['ceph', self.prefix()] + [p.mk_bash_example(simple) for p in self.params])
+        line = ' '.join(['ceph', self.prefix] + [p.mk_bash_example(simple) for p in self.params])
         return line
 
 
 class Sig:
     @staticmethod
-    def _param_to_sig(p):
+    def _parse_arg_desc(desc):
         try:
-            return {kv.split('=')[0]: kv.split('=')[1] for kv in p.split(',') if kv}
-        except IndexError:
-            return p
+            return dict(kv.split('=') for kv in desc.split(',') if kv)
```

Comment:
Aha, so then if we want to clean trailing whitespaces, then we have to:
```suggestion
            return dict(map(str.strip, kv.split('=')) for kv in desc.split(',') if kv)
```

Reply:
thanks. incorporated.